gflownet/gflownet.py

```diff
@@ -765,13 +931,31 @@ def test(self, **plot_kwargs):
         Computes metrics by sampling trajectories from the forward policy.
         """
         if self.buffer.test_pkl is None:
-            return self.l1, self.kl, self.jsd, (None,), {}
+            return (
+                self.l1,
+                self.kl,
+                self.jsd,
+                self.corr_logp_rewards,
+                self.nll_tt,
+                (None,),
+                {},
+            )
         with open(self.buffer.test_pkl, "rb") as f:
             dict_tt = pickle.load(f)
             x_tt = dict_tt["x"]
+
+        # Compute correlation between the rewards of the test data and the log
+        # likelihood of the data according the the GFlowNet policy; and NLL.
+        # TODO: organise code for better efficiency and readability
+        logprobs_x_tt = self.estimate_logprobs_data(x_tt, n_trajectories=10)
```

Comment:
Is there some empirical data indicating that 10 trajectories are sufficient for good estimation? Also, it would be nice to have this as an argument that can be changed from config.

Reply:
To your question: No, that I am aware of.

Agree with the _request_, so done [here](https://github.com/alexhernandezgarcia/gflownet/pull/167/commits/567533529dfe76a74519817df941f4cd65cf6680).